examples/plot_udct.py

```diff
@@ -0,0 +1,48 @@
+"""
+Uniform Discrete Curvelet Transform
+===================================
+This example shows how to use the :py:class:`pylops.signalprocessing.UDCT` operator to perform the
+Uniform Discrete Curvelet Transform on a (multi-dimensional) input array.
+"""
+
+import numpy as np
+from ucurv import udct, zoneplate, ucurvfwd, ucurvinv, ucurv2d_show
+import matplotlib.pyplot as plt
+import pylops
+plt.close("all")
+
+sz = [512, 512]
```

Comment:
You should try to write the entire example using the pylops.signalprocessing.UDCT methods (matvec/rmatvec) instead of those in the ucurv library. Of course, you can for example use the `ucurv2d_show` method but when you compute the forward and backward of your transform you should do this using the pylops operator.